lms/templates/courseware/course_about.html

```diff
@@ -319,6 +319,14 @@
             </li>
           % endif
 
+          % if pre_requisite_course:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon icon-list-ul"></i>
+            <p class="important-dates-item-title">${_("Requirements")}</p>
```

Comment:
@marcotuts ^^

Also, as Zia pointed out, the "Prerequisites" heading in the body of the About page comes from the default HTML in the **Course Overview** box on the **Schedule & Details** page. Can we change that default HTML so that it says "Requirements/Add information about the skills and knowledge students need to take this course" instead of "Prerequisites/Add information about course prerequisites here"?

![screenshot-229](https://cloud.githubusercontent.com/assets/3493253/5648942/65161d80-9660-11e4-968c-605724178a0e.png)